test/others/criu-ns/run.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
+
+import os
+import pty
+import shutil
+import subprocess
+import sys
+import time
+
+
+CRIU_BIN = "../../../criu/criu"
+CRIU_NS = "../../../scripts/criu-ns"
+IMG_DIR = "dumpdir"
+PIDFILE = "pidfile"
+
+
+def check_dumpdir():
+    if os.path.isdir(IMG_DIR):
+        shutil.rmtree(IMG_DIR)
+    os.mkdir(IMG_DIR, 0o755)
+
+
+def create_pty():
+    fd_m, fd_s = pty.openpty()
+    return (os.fdopen(fd_m, "wb"), os.fdopen(fd_s, "wb"))
+
+
+def create_isolated_dumpee():
+    with open("running", "w") as file:
+        file.name
+    fd_m, fd_s = create_pty()
+    pid = os.fork()
+    fd_m.close()
+    if pid == 0:
+        os.setsid()
+        os.dup2(fd_s.fileno(), 0)
+        os.dup2(fd_s.fileno(), 1)
+        os.dup2(fd_s.fileno(), 2)
```

Comment:
We should also make this terminal a controlling terminal of a dumpee process.

I have previously pointed you to an example https://github.com/Snorch/criu/blob/warasadura-criu-ns-test-v3/test/others/criu-ns/run.py#L36

Reply:
@Snorch when I make this terminal a controlling terminal by including the following line, `fcntl.ioctl(fd_s.fileno(), termios.TIOCSCTTY, 1)` I'm getting the following error,
```
Traceback (most recent call last):
  File "criu/test/others/criu-ns/../../../scripts/criu-ns", line 335, in <module>
    res = wrap_dump()
          ^^^^^^^^^^^
  File "criu/test/others/criu-ns/../../../scripts/criu-ns", line 303, in wrap_dump
    set_mntns(pid)
  File "criu/test/others/criu-ns/../../../scripts/criu-ns", line 273, in set_mntns
    ns_fd = os.open('/proc/%s/ns/mnt' % tpid, os.O_RDONLY)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/proc/4/ns/mnt'

```

Reply:
I was able to fix this issue.